staff-feedback/M1.md

```diff
@@ -0,0 +1,118 @@
+
+For this part, you received 15 points out of a maximum of 15.
+
+## UI
+
+We checked whether the elements of your app's UI are consistent (fonts, colors, etc.) and whether the layout is responsive (i.e., adapts to different screen sizes and orientations).
+
+We can't reach the login button when rotating the screen.
+
+For this part, you received 2 points out of a maximum of 5.
+
+# Quality & Delivery
+
+## Automated Testing
+
+We looked at whether you configured your CI to integrate with Sonar, whether your CI passes, and of course whether your CI automatically runs the test suite. Then we checked the statement coverage achieved by your test suite. We also looked at whether you have at least one meaningful UI test per screen, one meaningful unit test per repository, one meaningful unit test per view-model, and one meaningful end-to-end test.
+
+No end-to-end test present
+
+For this part, you received 28 points out of a maximum of 33.
+
+## Automated Delivery
+
+We checked whether you have set up your CI to automatically build the APK, and it does so successfully. We also wanted to see if the APK could be installed and run on a real device.
+
+
+
+For this part, you received 7 points out of a maximum of 7.
+
+## End-to-End Application Behavior
+
+We then interacted with your app manually, this was fun. We flagged any problems we encountered, such as crashes or bugs, and we classified the bugs into minor vs. major bugs.
+
+We cannot login into the application.
+
+For this part, you have been penalized -1 points.
+
+# Scrum
+
+## Documents
+
+The main things we looked for was whether you documented your team Retrospective and Stand-Up in a timely and thorough manner.
+
+Tthe Scrum Board is not empty, it is used, but not on time.
+
+For this part, you received 10 points out of a maximum of 10.
+
+## Backlogs & Planning
+
+We analyzed your product backlog, Sprint 3 backlog, and the Scrum board. For the PB, we checked whether it was up-to-date and whether you had populated it with user stories for the first epic and for a second epic. For the Sprint 3 backlog, we looked at whether it was up-to-date and whether the task descriptions were clear. We looked at whether tasks were assigned, whether they had a time estimate, and whether they had a priority. Finally, we evaluated whether your
+Scrum board was up-to-date and ready for Sprint 3.
+
+Sprint 3 backlog was not ready for the meeting
+The description is missing from almost all tasks at the time of the meeting, in the same manner, the task are not assigned, don't have time estimate nor priority.
+
+For this part, you received 8 points out of a maximum of 20.
+
+Based on the above points, your intermediate grade for this milestone M1 is 4.95. If you are interested in how this fits into the bigger grading scheme, please see [project README](https://github.com/swent-epfl/private/blob/main/project/README.md) and the [course README](https://github.com/swent-epfl/public/blob/main/README.md).

```

Comment:
The link to the project README is in the private repo so we don't have direct access to it.